Address all reviewers comments

MichalisPanayides opened the pull request:
## Comments addressed on this PR so far:
- Change vocabulary to be more application-focused ✔️
- Restructure the entire paper ✔️
- Add diagram of the game ✔️
- Define normal form game explicitly ✔️
- Address comment 5 by reviewer 3 ✔️
- Truncation effect ✔️
- Create response to reviewers ✔️

MichalisPanayides committed:
Prepare for addressing reviewers comments (#18)

* Add comments of reviewers

`main.md` consists of the comments of all three reviewers and the editor

* Include editor's comments

MichalisPanayides committed:
Change vocabulary to be more focused on the application (#33)

* Change vocabulary in game section

* Minor typo

Fix proselint failure by removing the word very from a comment

* Change vocabulary in methodology section

* Additional changes in game section

- Replace waiting zone 1 with hospital
- Replace waiting zone 2 with parking space
- Use type 2 patients everywhere instead of ambulance patients

* Change vocabulary in queueing section

MichalisPanayides committed:
Change overall structure (#35)

* Change the overall structure

New structure:
1. Intro + Motvation + Major cotnributions
2. Lit review + Major contributions
3. Queueing model (with performance measures formulas)
4. Game model (with methodology and numerical results)
5. Conclusion

* Minor changes

- Minor changes to sections and subsections
- Move figure from game section to queueing section

* Fix failing proselint task

Clean the text before running proselint on it. Currently just removes
`\centering' command from .tex files

* Modify introduction section

* Restructure game theoretic model section

* Minor changes on queueing section

* Change some of the definitions

* Clearly state the major contributions

* Apply reiview suggestions from Vince

MichalisPanayides committed:
Add diagram of the game (#36)

* Create a tikz image of the game

* Add minor changes to the text

* Update known words

* Remove the word "very"

* Fix caption

* Apply suggestions from Vince's review

Co-authored-by: Vince Knight <vince@vknight.org>

Co-authored-by: Vince Knight <vince@vknight.org>

MichalisPanayides committed:
Add changes to conclusion (#37)

* Add changes to conclusion

- Rewrite the major contributions of the paper
- Add closed-form of steady state as area for further work

* Apply changes from Vince's review

* Fix failing alex check

Instead of `just` use `only`

MichalisPanayides committed:
Define normal form game explicitly (#40)

MichalisPanayides committed:
Address comment 5 by reviewer 3 (#39)

* Add paragraph addressing reviewr's comment

* Update known words

MichalisPanayides committed:
Investigate the truncation effect (#34)

* Get initial plots of truncation effects

Plots added on unbounded simulation VS Markov chain (while
increasing capacity):
- Waiting time
- Blocking time
- Proportion of individuals within time

* Add bounded simulation to same plot

* Write script to generate duration for each method

* Get timing data when M=50

Times for N = (10, 30, 50, inf) for markov (waiting, blocking and
proportion formula) and simulation (1 trial, 20 trials)

* Get timing data when M=5

Times for N = (10, 30, 50, inf) for markov (waiting, blocking and
proportion formula) and simulation (1 trial, 20 trials)

* Create a table of the results in jupyter notebook

* Run jupyter notebook

* Add sentence to explain reason behind figures 4-6

* Modify truncation and timing scripts

* Add new data and plots to illustrate truncation

Data are now stored instead of having to rerun the script to get them
and the new plots have some changed labels

* Update plots in paper to illustarte the truncation

* Produce data for timing experiments

* Get the tabe of timings

* Minor changes to truncation paragraph

* Update known words

* Apply suggestion by alex

* Apply suggestions from Vince's review

Co-authored-by: Vince Knight <vince@vknight.org>

Co-authored-by: Vince Knight <vince@vknight.org>

MichalisPanayides committed:
Address several TODOs (#41)

* Update highlights

* Address several todos

* Add a final TODO to be fixed

* Add example of another application of the model

* Fix a typo

* Add script and notebook of previous plots

* Rename jupyter notebook

* Change plots axis labels

* Scale payoff matrices to be more presentable

* Remove unnecessary equations

* Minor additional changes

* Address the high traffic intensity

MichalisPanayides committed:
Response to reviewers (#38)

* Write initial response to reviewers

- Not all comments are addressed

* Apply suggestions from proselint

* Update known words

* Add minor change

* Apply suggestions from Vince's review

Co-authored-by: Vince Knight <vince@vknight.org>

* Minor changes

* Apply proselint suggestion

* MInor changes

* Address two more comments

* Address another two comments

- More applications examples
- Rewritten highlights

* Address the final comments

* Apply suggestions by proselint

Co-authored-by: Vince Knight <vince@vknight.org>

MichalisPanayides committed:
Compile PDF (#43)

drvinceknight commented on sections/conclusion/main.tex:
```diff
@@ -1,58 +1,82 @@
+blocked on a separate queue.
+An example of such setting can be any type of delivery service where customers
+can purchase goods either online or walk in the store.
+More specifically consider a food delivery service where customers can order
+food online or walk in the store.
```
```suggestion
```

(repetition)

drvinceknight commented on sections/conclusion/main.tex:
```diff
@@ -1,58 +1,82 @@
-This in turn can be used to model ED staff as agents each with their own 
+Another extension would be to explore the behaviour of the ED staff via an
+agent-based model.
+This in turn can be used to model ED staff as agents each with their own
 behavioural traits.
```
```suggestion
This in turn can be used to model emergent behaviour based on
assumptions of individual behavioural traits of ED staff.
```

drvinceknight commented on sections/model_overview/main.tex:
```diff
@@ -1,197 +1,163 @@
 individuals and smaller \(\alpha\) a higher weight on the time blocked. 
+Note here that the choice of \(p_A\) and \(p_B\) rely solely on equation
+\ref{eq:obj_distributor}.
+An extension that can be applied here is the use of unobservable queues.
+The ambulance service does not necessarily know at which state hospitals A and
+B are at, and thus may need to make decisions based on other
+information~\cite{unobservablequeue}.
```
I think this is a bit confused: currently the model **is** an unobservable model: everything is based on rates and expectations.

MichalisPanayides replied:
Oh, I think I misinterpreted this issue #25. Will change this accordingly

drvinceknight commented on sections/model_overview/numerical_results/main.tex:
```diff
@@ -293,81 +226,81 @@ \subsection{Numerical results}\label{sec:numerical_results}
 \ref{fig:ard_penalty}).
-Note that the values of the payoff matrices only differ on the fourth digit
-which is the raw utility of the strategies played.
+Note that for the presentation of data, a scaling has been applied to obtain
+the values of the payoff matrices.
+The results are not affected by this scaling.
```
```suggestion
Note that for the presentation of data, an affine transformation has been applied to obtain
the values of the payoff matrices.
The results are not affected by this scaling.
```

drvinceknight replied:
We should specifically write what the transformation is (in brackets somewhere, it's not super important).

MichalisPanayides committed:
Changes after first supervisory review (#44)

* Apply minor changes

* Highlight choice of queue and potential extensions

Rewrite the whole paragraph so that the choice of unobservable queues
is highlighted and clearly state the potential new extensions

* Replace reference with one that makes more sense

* Compile main.pdf

* Update known words

* Reword a sentence

MichalisPanayides committed:
Changes after second supervisory review (#45)

* Add changes to response to reviewers

* Fix square in general markov model image

* Add another keyword

* Apply changes to intro and lit review

* Apply changes to queueing mode section

* Apply changes to game theoretic mode section

* Apply changes to conclusion

* Minor changes to appendix

* Compile PDF

MichalisPanayides committed:
Generate pdf file with all changes

MichalisPanayides committed:
Rename some file names

MichalisPanayides committed:
Remove spaces from file name
